db/models.py

```diff
@@ -14,3 +14,39 @@ class Actor(models.Model):
 
     def __str__(self) -> str:
         return f"{self.first_name} {self.last_name}"
+
+
+class Movie(models.Model):
+    title = models.CharField(max_length=255)
+    description = models.TextField()
+    actors = models.ManyToManyField(Actor)
+    genres = models.ManyToManyField(Genre)
```

Comment:
The `ManyToManyField` fields `actors` and `genres` in the `Movie` model should have `related_name` attributes to define reverse relationships. Consider adding `related_name='movies'` or something similar to make the relationships clear and unique.